packages/manager/modules/pci/src/projects/project/storages/databases/database/pools/information/information.html

```diff
@@ -0,0 +1,86 @@
+<oui-modal
+    data-heading="{{:: 'pci_databases_pools_information_title' | translate }}"
+    data-on-dismiss="$ctrl.goBack()"
+>
+    <div class="mb-3 information">
+        <dl>
+            <div class="row my-2 py-2">
+                <dt
+                    class="col-sm-3 font-weight-bold"
+                    data-translate="pci_databases_pools_information_database"
+                ></dt>
+                <dd
+                    class="col-sm-9 my-auto"
+                    data-ng-bind="$ctrl.pool.database?.name"
+                ></dd>
+            </div>
+            <div class="row my-2 py-2 border-top">
+                <dt
+                    class="col-sm-3 font-weight-bold"
+                    data-translate="pci_databases_pools_information_port"
+                ></dt>
+                <dd
+                    class="col-sm-9 my-auto"
+                    data-ng-bind="$ctrl.database.port"
+                ></dd>
+            </div>
+            <div class="row my-2 py-2 border-top">
+                <dt
+                    class="col-sm-3 font-weight-bold"
+                    data-translate="pci_databases_pools_information_ssl_mode"
+                ></dt>
+                <dd
+                    class="col-sm-9 my-auto text-capitalize"
+                    data-ng-bind="$ctrl.database.sslMode"
+                ></dd>
+            </div>
+            <div class="row my-2 py-2 border-top">
+                <div class="font-weight-bold copy-clipboard ml-3">
+                    <span
+                        data-translate="pci_databases_pools_information_certificate_ca"
+                    ></span>
+                    <span
+                        ng-click="$ctrl.downloadCertificate(database)"
```

Comment:
**issues(blocking):** Why not prefer using a regular `button`? Looks strange to have an `ngClick` attached to a `<span>` element. This leads to accessibility issue.